app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2>Sign up</h2>
+
+<div class="flex-col form-logs">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo %><br />
+    <%= f.text_field :photo, autofocus: true, autocomplete: "https://" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_area :bio, autofocus: true, autocomplete: "bio" %>
+  </div>
+
+  <div class="field"> 
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+</div>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] The registration screen input fields are not aligning well.  Look:

![image](https://user-images.githubusercontent.com/57421823/173619332-bbd3d2c0-7a1e-4a97-bdb1-55096a419c25.png)

Please, kindly correct this issue so the registration page looks professional 💪 

- [ ] OPTIONAL: I highly suggest your make the registration screen more in line with the login screen general design presented in the log-in wireframe.